python-lib/google_translate_api_formatting.py

```diff
@@ -92,16 +71,18 @@ def __init__(
         error_handling: ErrorHandlingEnum = ErrorHandlingEnum.LOG,
     ):
         super().__init__(input_df, column_prefix, error_handling)
-        self.translated_text_column_name = "{input_column}_{language}".format(input_column=input_column, language=target_language.replace('-','_'))
+        self.translated_text_column_name = generate_unique(
```

Comment:
I think we need to do a similar thing for the "detected_source_language" column:
- Generate a dynamic column name, perhaps "{col}_detected_language" and make it unique
- Add a column description

Reply:
... and thanks, I appreciate the inclusion of `generate_unique` 💯